### DISCUSSION TITLE
Downgrade to Big Sur

### DISCUSSION THREAD

**Discussion by controlfix87** [osy86/HaC-Mini#968] [Q&A] [answered]
Hi,
I've updated to macOS monteray, is there a way to downgrade to MacOS BigSur? Without installing everything again?

**Comment by osy** [comment, marked as answer]
It is the same as with a Mac. You need to make a backup (Time Machine or something), format, reinstall, and manually restore your files. Apple doesn't make downgrading easy. That's why I always keep a backup before upgrading.

**Reply by Medizinmann-MD** [reply]
I can recommend Carbon Copy!

And making a 1:1 copy before any Update/Upgrade!

**Comment by MokOopsing** [comment]
@osy What's the lowest version that hac-mini support?

**Reply by osy** [reply]
macOS 10.14.5

**Reply by MokOopsing** [reply]
> macOS 10.14.5

If I downgrade to 10.14.5, which version of hac-mini is the best choice?  how about v2.9?

**Reply by osy** [reply]
v3.1.4
